scripts/.phantom-run-tests.js

```diff
@@ -0,0 +1,41 @@
+var page = require('webpage').create();
```

Comment:
Any reason not to drag this into the future? Make this and `logToken` into `const`s, `let someTestsFailed`...